### PR TITLE
[ISSUE #4602] when wechat send message api response errcode is not zero, the wechat sink connector does not throw IllegalAccessException

### LINKED ISSUE #4602: [Bug] when wechat send message api response errcode is not zero, the wechat sink connector does not throw IllegalAccessException
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Environment

Windows

### EventMesh version

master

### What happened

when i use wechat-sink-connector send message test some abnormal case, like this
image
when response errcode is not zero , it should throw IllegalAccessException 
image
actually it does not, because the class TemplateMessageResponse.calss filed errocode is wrong, not match the wecaht response json errcode
image
sorry for missing  abnormal case in unit test, i will add  abnormal case in unit test and fix the bug

### How to reproduce

 i use wechat-sink-connector send message test some abnormal case, make the wechat server response errcode is not zero

### Debug logs

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct) *

### PR DESCRIPTION
**Pull request by wizardzhang**
Fixes #4602

### Motivation
fix  bug when wecaht api response fail

### Modifications

rename org.apache.eventmesh.connector.wechat.sink.connector.TemplateMessageResponse#errocode
to org.apache.eventmesh.connector.wechat.sink.connector.TemplateMessageResponse#errcode
add abnormal test case

### CODE REVIEW

File: eventmesh-connectors/eventmesh-connector-wechat/src/test/java/org/apache/eventmesh/connector/wechat/sink/connector/WeChatSinkConnectorTest.java [hunk 1]
```diff
@@ -95,31 +109,43 @@ public void setUp() throws Exception {
...
 
+        RecordPartition partition = new RecordPartition();
+        RecordOffset offset = new RecordOffset();
+        List<ConnectRecord> records = new ArrayList<>();
+        ConnectRecord connectRecord = new ConnectRecord(partition, offset,
+            System.currentTimeMillis(), "Hello, EventMesh!".getBytes(StandardCharsets.UTF_8));
+        records.add(connectRecord);
         weChatSinkConnector.put(records);
-        verify(okHttpClient, times(times + 1)).newCall(any(Request.class));
+        verify(okHttpClient, times(2)).newCall(any(Request.class));
```
[the remark this answers is not in the document]
Comment (wizardzhang) [reply eventmesh-connectors/eventmesh-connector-wechat/src/test/java/org/apache/eventmesh/connector/wechat/sink/connector/WeChatSinkConnectorTest.java:148]: any idea for test private method， i konw powermock can test private method, it is a good idea to use it? @pandaapo

**Comment by wizardzhang** [comment]
> You can add this Sink Connector to the list in this document. https://github.com/apache/eventmesh/tree/master/eventmesh-connectors#connector-status

already done, please review

File: eventmesh-connectors/eventmesh-connector-wechat/src/test/java/org/apache/eventmesh/connector/wechat/sink/connector/WeChatSinkConnectorTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (wizardzhang) [reply eventmesh-connectors/eventmesh-connector-wechat/src/test/java/org/apache/eventmesh/connector/wechat/sink/connector/WeChatSinkConnectorTest.java]: ok, i choose to use JUnit's assertThrows()

**Review by mxsm — APPROVED** [review]
LGTM

**Comment by wizardzhang** [comment]
> Finally, are you willing to write a document for your connector ([#4601 (comment)](https://github.com/apache/eventmesh/issues/4601#issuecomment-1838736726)) to facilitate user understanding and use. If you are willing, you can do it in this PR or in a new PR in the future.
> 
> 最后，请问是否愿意给您的这个Connector写一下文档 （[#4601 (comment)](https://github.com/apache/eventmesh/issues/4601#issuecomment-1838736726) ），方便用户理解和使用。如果您愿意，可以在该PR中做，也可以以后在新的PR中做。

i prefer in a new PR to do this

**Review by Pil0tXia — APPROVED** [review]
LGTM

**Comment by wizardzhang** [comment]
@pandaapo  hello, i found  some commit in this PR used wrong email, so i want to close this PR, an create a new one

**Pull request closed without merging**